src/taskgraph/run-task/run-task

```diff
@@ -1078,6 +1084,51 @@ def _display_python_version():
         b"Python version: "+ bytes(platform.python_version(), 'utf-8')
     )
 
+def _display_python_version_in_sub_process(subprocess: subprocess):
+    print_line(
+        b"setup",
+        b"pyhton --version: ",
+    )
+    try:
+        print_line(
+            b"setup",
+            subprocess.check_output(["python", "--version"])
```

Comment:
Improvement: Let's add the `--version` argument a second time. This will print out more debug data. E.g.: 

```
>>> subprocess.check_output(["python", "--version"])
b'Python 3.11.3\n'

>>> subprocess.check_output(["python", "--version", "--version"])
b'Python 3.11.3 (main, Apr  7 2023, 20:13:31) [Clang 14.0.0 (clang-1400.0.29.202)]\n'
```

I'd recommend adding a code comment to highlight to future readers this is intentional.